src/components/Modals/RequirementOptionsModal.vue

```diff
@@ -157,6 +157,7 @@
                 } else {
                     if (!this.onSelectionBar) {
                         this.course.inRequirementBar = true;
+                        this.course.overridden = false;
```

Comment:
need to also hide major and minor chip in the requirement bar. Reset them to default

Reply:
We don't store the chip info there. By setting `inRequirementBar = true` we effectively do that.

Reply:
A lot of the cases you want address are in the other PR.

Reply:
sounds good